src/i18n.ts

```diff
@@ -1,22 +1,18 @@
-import i18next, {
-  i18n,
-  InitOptions
-} from "i18next";
+import i18next from "i18next";
 import {
   DOM,
   PLATFORM
 } from "aurelia-pal";
 import * as LogManager from "aurelia-logging";
 import { EventAggregator } from "aurelia-event-aggregator";
 import { BindingSignaler } from "aurelia-templating-resources";
-import { autoinject } from "aurelia-framework";
 
-export interface AureliaEnhancedOptions extends InitOptions {
+export interface AureliaEnhancedOptions extends i18next.InitOptions {
```

Comment:
It's just for reducing amount of manual work related to reduce the amount of manual work related to generated `d.ts`. a single default import instead of a bunch of import. Also, personally, it helps communicate better where those interface comes from